src/chart-addons/stacked-legend.js

```diff
@@ -63,6 +63,21 @@ function setColorScaleDomain(domain) {
   }
 }
 
+function setColorScaleDomain_v1(domain) {
```

Comment:
I think we should make the object structure same at some point later for raster charts in immerse and the charting. Needs a minor refactoring.

Reply:
absolutely. Not sure why that wasn't done. It's confusing to use two different schemas. Apparently it diverged with heatmap but wasn't brought in line with the other raster layer types. This requires a refactor. See my last comment in this ticket: https://github.com/mapd/mapd-immerse/issues/3703